packages/app/components/stack-modal/StackOrdersTable.tsx

```diff
@@ -79,18 +79,22 @@ export const StackOrdersTable = ({ stackOrder }: StackOrderProps) => {
   );
 };
 
-const TableCowBody = ({ cowOrders }: { cowOrders: CowOrder[] }) => {
-  const chainId = 100; // @todo change it to useNetwork wen sdk
-  const formatAmount = (amount: string) =>
-    convertedAmount(amount, 18).toFixed(4);
+const TableCowBody = ({
+  stackOrder,
+  cowOrders,
+}: {
+  stackOrder: StackOrder;
+  cowOrders: CowOrder[];
+}) => {
+  const chainId = 100; // @todo use context or useNetwork hook
```

Comment:
This seems to be missing

Reply:
Also, where is `chainId` being used?

Reply:
`cowExplorerUrl` function bellow

Reply:
we're hardcoding this, we have to remove ethereum from the networks for now.

Reply:
We should do it on other PR.

Created a task for that
https://linear.app/swaprdev/issue/STK-101/remove-ethereum-chain-from-networks-and-remove-hardcoded-chainids